src/markdown-pages/highlights.md

```diff
@@ -11,13 +11,28 @@ Post a question or share your experiences with other users in our <a href="https
 
 Access the Galasa source code in [GitHub](https://github.com/galasa-dev) and open issues in the [project management repository](https://github.com/galasa-dev/projectmanagement).
 
-## 0.24.0 - Release Highlights
+## 0.25.0 - Release Highlights
+
+
+-	Colour support for 3270 application streams. Tests can now validate that an application is using the correct colours and highlighting for specified field(s) and screen position.
+-	Various defect fixes, including fixing broken LTS connections in Java 11, updates to 3270 modifiable fields, and updates to the REST API.
+-	VTP Manager enhancements, including additional logging for problem diagnosis and increased keyboard wait times to minimise the risk of keyboard locking.
+-	New features added to the galasactl command-line tool around logging. You can turn logging off or send log information to a specified file or to the console. If logging is turned on, errors now appear on stderr and in the log. Syntax documentation is fully generated and linked to from the <a href="https://github.com/galasa-dev/cli" target="_blank"> README</a> in the CLI repository.
+-	Errors are numbered, for example, `GAL1028E: Failed to unmarshal test catalog from REST reply for property 'xxx'`
```

Comment:
These are all sub-bullets of the galasactl changes really. Maybe we need a `galasactl` bullet to put them all under ?